tests/JsonFieldTest.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+class JsonFieldTest extends FunctionalTest
```

Comment:
May as well make this SapphireTest - doesn't look like this test classes uses anything controller related